async-aggregators/src/main.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from typing import Any
+
+from PyQt6.QtWidgets import QApplication
+
+from src.level import Level
+from src.window import Window
+
+
+def parse_level_input(args: Any) -> int:
+    """
+    Parse the level input
+
+    :param args: arguments
+    :return:
+    """
+    max_level = 5
+    if len(args) == 0:
+        level = 1
+    elif len(args) == 1:
+        try:
+            level = int(args[0])
+        except ValueError:
+            raise ValueError("First argument is not an integer")
+    else:
+        raise ValueError("Too many arguments specified")
+    if level < 1 or level > max_level:
+        raise ValueError(
+            f"Invalid level number. Please pick a number between 1 and {max_level}"
+        )
```

Comment:
Your tox file specifies 119 as the col limit, it's good for everyone to be on the same page for formatting style decisions.
```suggestion
        raise ValueError(f"Invalid level number. Please pick a number between 1 and {max_level}")
```